src/cascadia/Remoting/WindowManager.cpp

```diff
@@ -101,12 +102,20 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
         // monarch, and see what happens here.
 
         bool proposedCommandline = false;
-        Remoting::ProposeCommandlineResult result{ nullptr };
+        winrt::com_ptr<implementation::ProposeCommandlineResult> result{ nullptr };
         while (!proposedCommandline)
         {
             try
             {
-                result = _monarch.ProposeCommandline(args);
+                // MSFT:38542548 _We believe_ that this is the source of the
+                // crash here. After we get the result, stash it's values into a
+                // local copy, so that we can check them later. If the Monarch
+                // dies between now and the inspection of
+                // `result.ShouldCreateWindow` below, we don't want to explode
+                // (since _proposeToMonarch is not try/caught).
+                Remoting::ProposeCommandlineResult outOfProcResult = _monarch.ProposeCommandline(args);
+                result = winrt::make_self<implementation::ProposeCommandlineResult>(outOfProcResult);

```

Comment:
You can reduce the diff here by using `winrt::make`; it will return the projected type, so you don't need to switch to `com_ptr` or from `.` to `->`

It would also be easier to cherry pick that way 😁